app/VERSION.txt

```diff
@@ -0,0 +1 @@
+1.26.0
```

Comment:
@vishwasbabu sorry for the late update. Yes, VERSION.txt was replaced by release.json but the file was not removed. Will do a separate change for this.